governance/dashboard-update.py

```
@@ -0,0 +1,292 @@
+    names = [r["name"] for r in repos if r.get("status") == "active"]
+    if not names:
+        raise Infra("REPOS.yaml 无 active 仓")
+    return names
+
+
+def _iso(s):
+    return _dt.datetime.fromisoformat((s or NOW.isoformat()).replace("Z", "+00:00"))
+
+
+def scan_cards(repos):
+    """与 board-sync.py 同判据的独立轻量扫描（自包含；真相源=issue label）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)